tests/xdp/packages.config

```diff
@@ -1,8 +1,4 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<!--
-  Copyright (c) Microsoft Corporation
-  SPDX-License-Identifier: MIT
--->
 <packages>
```

Comment:
why was this file moved?

Reply:
Goal is to get rid if it, but this test has a hard dependency on Catch 2.8.0

Reply:
which part? The custom argument? Please create an issue suggesting alternate so that the reference can be removed.

Reply:
https://github.com/microsoft/ebpf-for-windows/issues/820